examples/sqlalchemy/orm.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy import create_engine, Column, DateTime, String
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+
+Base = declarative_base()
+
+
+class Pet(Base):
+    __tablename__ = 'pets'
+    id = Column(String(20), primary_key=True)
+    name = Column(String(100))
+    animal_type = Column(String(20))
+    created = Column(DateTime())
+    
+    def update(self, id=None, name=None, animal_type=None, tags=None, 
+            created=None):
+        if name is not None:
+            self.name = name
+        if animal_type is not None:
+            self.animal_type = animal_type
+        if created is not None:
+            self.created = created
+        
+    def dump(self):
+        return dict([(k,v) for k,v in self.__dict__.items() if k[0] != '_'])
```

Comment:
You can replace `self.__dict__` with `vars(self)` to avoid explicitly getting a dunder property.